react/features/recording/components/LiveStream/AbstractLiveStreamButton.js

```diff
@@ -109,6 +109,10 @@ export default class AbstractLiveStreamButton<P: Props>
 export function _mapStateToProps(state: Object, ownProps: Props) {
     let { visible } = ownProps;
 
+    // a button can be disabled/enabled only if enableFeaturesBasedOnToken
+    // is on
+    let disabledByFeatures;
```

Comment:
Why change this so it's not defaulted to false?

Reply:
Okay, I think part of the confusion is there might be a bug with buttons. Seems like `visible` isn't being respected. I'm checking AbstractLiveStreamButton, AbstractButton, LiveStreamButton.web, LiveStreamButton.native, and I don't see where `visible` gets used.

Reply:
Nevermind, I tracked it down to ToolboxItem -> AbstractToolboxItem

Reply:
So I think the bug is here:
```  
if (typeof visible === 'undefined') {
        visible = interfaceConfig.TOOLBAR_BUTTONS.includes('livestreaming')
            && (abstractProps.visible || _liveStreamDisabledTooltipKey);
  }
```
Let's say `interfaceConfig.TOOLBAR_BUTTONS.includes('livestreaming')` is true. Let's say `abstractProps.visible` is `abstractProps.visible` is false. Then what's returned is `_liveStreamDisabledTooltipKey` which is undefined. There's a default prop in AbstractButton that makes visible equal true.